setup.cfg

```diff
@@ -22,13 +22,27 @@ classifiers =
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
 
 [options]
 package_dir =
     = src
 packages = find_namespace:
 install_requires =
-    stactools == v0.2.1a1
+    stactools
```

Comment:
install_requires can't specify a single version. Otherwise this plugin might break others within the stactools namespace.

Reply:
I think the best approach will be to follow https://packaging.python.org/en/latest/discussions/install-requires-vs-requirements/#install-requires-vs-requirements-files. Stactools plugin packages will require just an abstract `stactools` (no upper version) and then applications such as this project's docker images will use a requirements.txt (or lockfile) to specify a known good set of Python packages.

Reply:
Hm, that's interesting, I hadn't seen that guidance before. While it seems a little non-DRY, I'm not necessarily opposed. It would be a bit of a ecosystem change, as we've been relying on versions as specified in `setup.cfg` up to this point.

For this specific package, whatever works should be fine. If we wanted to make the change at a **stactools** level, I think a RFC there would be appropriate.